core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -120,15 +122,15 @@ oppia.factory('AssetsBackendApiService', [
         assetType) {
       if (_isAssetCurrentlyBeingRequested(filename, ASSET_TYPE_AUDIO)) {
         for (var index = 0; index <
-             _audioFilesCurrentlyBeingRequested.length; index++) {
```

Comment:
Nit: why change this indentation? I think it was correct before. (also below)

Reply:
changed to before..!